libqtile/scratchpad.py

```diff
@@ -159,6 +160,8 @@ def __init__(self, window, scratchpad_name, ddconfig):
         self.y = ddconfig.y
         self.width = ddconfig.width
         self.height = ddconfig.height
+        # Let's add the window to the scratchpad group.
+        window.togroup(scratchpad_name)
```

Comment:
What effect does this have on behaviour?

Reply:
Ideally this code is called, when the scratchpad is first created or when the state is restored. We explicitly show/hide based on the state in both the cases. So without these lines, it would still work. But, I thought this is good initial state when we initialize a dropdown.

Reply:
Without it, does it mean that `window.group` is `None` until the window is shown for the first time? And then from then on, `window.group` is either the scratchpad group or the current visible group? (Just trying to understand). If that's the case then yeah I agree, it makes more sense for it to never be `None` and to get the correct initial state.